lib/elements/dom-repeat.js

```diff
@@ -471,6 +472,12 @@ export class DomRepeat extends domRepeatBase {
       this.observe.replace('.*', '.').split(' ');
   }
 
+  __initialCountChanged(initialCount) {
+    // When enabling chunking, start the limit at the current rendered count,
+    // otherwie ensure the list is unlimited
+    this.__limit = initialCount ? this.renderedItemCount : Infinity;
```

Comment:
Setting of `__limit` here makes this hard to grok. Consider instead moving this to `__updateLimit`. If a test needs to manually modify `__limit` perhaps it can also modify the behavior of `__updateLimit` as needed.

Reply:
Done.  Limit is now derived locally from existing state and no longer itself a state variable.  Good improvement.